NinchatSDK/WebRTC/NINWebRTCClient.m

```diff
@@ -535,9 +545,13 @@ -(BOOL) enableLocalVideo {
 
 -(void) peerConnection:(nonnull RTCPeerConnection *)peerConnection didAddStream:(nonnull RTCMediaStream *)stream {
     NSLog(@"WebRTC: Received stream %@ with %lu video tracks and %lu audio tracks", stream.streamId, (unsigned long)stream.videoTracks.count, (unsigned long)stream.audioTracks.count);
-
+    
```

Comment:
There are lots of "christmas trees" appearing lately on logs.

Reply:
Sorry, I didn't got your meaning. What are "Christmas Trees"? :D

Reply:
Sorry my bad.. I meant whitespaces at the end of lines (or lines containing only whitespaces). :)

Reply:
Oh, OK. Actually the lines help me to organize the code in the editor with a cleaner manner. This is just a habit to have an intent in the code to make it easier to collapse different sections.

Reply:
Weird editor, but ok :)